logstash-core/lib/logstash/config/modules_common.rb

```diff
@@ -72,7 +72,15 @@ def self.pipeline_configs(settings)
           # Only import data if it's not a config test and --setup is true
           if !config_test && modul_setup
             esclient = LogStash::ElasticsearchClient.build(module_hash)
-            kbnclient = LogStash::Modules::KibanaClient.new(module_hash)
+
+            kbn_settings_hash = Hash.new
+            module_hash.each do | setting_name, setting_value |
```

Comment:
@ycombinator if I understand this correctly, this is to make existing settings backward-compatible.
Do we actually need to be this careful here? (probably more of a question for @andrewvc ?)

Reply:
Actually its just more to decouple the Kibana client setting names from module setting names. Module settings, by the time they get to this point in the code, are named like `var.kibana.ssl.verification_mode`. These were being passed in to the Kibana client as-is for initialization.

I thought the Kibana client implementation should no longer be so tightly coupled to modules. It really doesn't need to take `var.kibana.ssl.verification.mode` when it could do with just `ssl.verification.mode`.

Looking at this differently, why would another consumer of this client (say X-Pack Management)  want to set `var.kibana.ssl.verification.mode`, that is, start all the settings for this Kibana client with `var.kibana.`?

Reply:
@ycombinator I think I didn't explain myself very well here. My question was more about whether or not we want to keep supporting the `var.kibana` prefix here in the first place and not just break compatibility. I do like the renaming :)

That said ... on second thought, that was a stupid point probably :) I'm 100% on board with this. @andrewvc feel free to ignore the above :)

Reply:
Oh I see. I think the `var.kibana` prefix comes from modules because they can have other vars too. For instance, the Netflow module also has `var.input.udp.port`. I imagine we want to keep the prefixing in modules for disambiguation.